partner_event/views/event_registration_view.xml

```diff
@@ -4,6 +4,26 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
 
+<record id="view_event_registration_form" model="ir.ui.view">
+    <field name="model">event.registration</field>
+    <field name="inherit_id" ref="event.view_event_registration_form"/>
+    <field name="arch" type="xml">
+        <field name="name" position="after">
+            <field name="attendee_partner_id"/>
```

Comment:
I can't understand why the `attendee_partner_id` is taking more relevance than the `partner_id` with this PR, I feel like this changes a working behavior and it's giving more relevance to a field that actually does not keep all the partner's logic.

I do understand that the attendee_partner_id could be different from the partner_id. But if the partner_id is the one paying, why would the attendee_partner_id has to be more relevant?

Reply:
I also feel this causes 'billing inconsistency' for a company that their core business and billing is based on events, as a partner shouldn't have a billing that differs from their event registration.

Reply:
@gustavovalverde your concern is that ``attendee_partner_id`` should not be introduced?
What is the case when attendee shall be the same as person  that paid the ticket? It limits the sale, doesn't?

Reply:
I don't see any problem, as they are different fields.

Reply:
I'm no against of having the `attendee_partner_id` , but just of making this field *more relevant* (in a functional way) than the `partner_id` makes it a bit harder for analysis of what's going on.

For example: 
![image](https://user-images.githubusercontent.com/16890942/38248091-46a68340-3715-11e8-90c0-5618a8dbb72d.png)

It's more common a company sending their employees (partner_childs_id), than a company serving as a middleman (this could be around a 10%), and only in the latter case the `attendee_partner_id` is more relevant.

In v8 it was taking the events count from the partner (which makes sense as an SO can only have 1 partner), now we saw the analytics and it was distributing the event count to the attendees.

Summarizing: I'm not saying to remove this field, It's just that for most uses cases that I've worked on in the events and education industry this field is just not more relevant than the partner_id.

Reply:
More relevant in which part exactly? For mass mailings, it's more relevant to send the event information to the contact than to the company.

Reply:
For mailing lists I already have the Attendee Name, their Phone and Email. Having a partner for the attendee is not needed for that specific case.

This could be even marked as a "taste" thing. For my clients, I just removed the field from the view in the registration (so it wouldn't affect the event count). I just wanted to make the __heads_up__.

I'll add this to my work pipe, to consider a PR here, where this behavior can be 'configurable':
- Relate event based on Attendee Partner
- Relate event based on Partner

Reply:
@gustavovalverde 

>  can't understand why the attendee_partner_id is taking more relevance than the partner_id with this PR, I feel like this changes a working behavior and it's giving more relevance to a field that actually does not keep all the partner's logic.

> I do understand that the attendee_partner_id could be different from the partner_id. But if the partner_id is the one paying, why would the attendee_partner_id to be more relevant?

Why more relevance? This doesn't mean more relevance mean each one has relevance.

- One is the the res.partner who pay
- Two is the res.partner who attend
- One and two could be equal or not
- One has relevance for Sale, Invoice purpose
- Second has relevance for Event analysis purpose

> For mailing lists I already have the Attendee Name, their Phone and Email. Having a partner for the attendee is not needed for that specific case

Yes, is needed. Is basic. Because your Partners (companies and contacts) are your customer you manage this in you CRM and this module make this, relate you Partners (CRM) with you Event management.

@gustavovalverde if you don't understand or need it, just don't use this module because if understand you, what you expect is what Odoo makes by default. This module is to improve and add functionality.

Please go to Odoo  (without this module) and let me know all your res.partner that has attendee to one single event. You cannot. You can only tell my which partners has paid and event. If you want to know which partners has attended to an event then you need this module.

Note also that you could have 1 sale order and 1 partner who paid but 10 different attendees (which are also partners)

Hope this helps